plugins/GSdx/Window/GSSetting.cpp

```diff
@@ -196,10 +196,10 @@ const char* dialog_message(int ID, bool* updateText) {
 			return "By default, the texture cache handles partial invalidations. Unfortunately it is very costly to compute CPU wise."
 				"\n\nThis hack replaces the partial invalidation with a complete deletion of the texture to reduce the CPU load.\n\nIt helps snowblind engine games.";
 		case IDC_LARGE_FB:
-			return "Allocate a large framebuffer to be compliant with GS memory (Prevents FMV flickering).\n"
-				"It increases GPU/memory requirements.\n\n"
-				"Note: It should be disabled for Armored Core, Destroy All Humans, and Gran Turismo.\n"
-				"It will amplify RAM/VRAM spikes. Other games might be affected as well.";
+			return "Reserves a bigger sized framebuffer for textures to prevent FMV flickers.\n"
+				"Increases GPU/memory requirements but also easier to hit the limit with lower FPS.\n"
+				"Enabling this can amplify stuttering due to low RAM/VRAM.\n\n"
+				"Note: It should be disabled (especially for Armored Core, Destroy All Humans, Gran Turismo and many others).";
```

Comment:
Likewise here, this just makes it seem that user should never turn this option on which isn't the case.
Option is a lot more useful.

Reply:
I'll just drop 'many' and make the language not as forceful for not enabling it. In the previous tooltip it was like you should always enable it, i even did that for years before contributing.